popup.js

```diff
@@ -86,7 +86,7 @@ var n = noty({
 
 document.addEventListener("DOMContentLoaded", function() {
   var download_btn = document.getElementById("download_btn");
-  download_btn.addEventListener("click", function() {
+     download_btn.addEventListener("click",addDownload);
```

Comment:
@agentmilindu The brackets are not closing here, I actually authored this addDownload piece of code yesterday (Rohan didn't merge it because he had already moved ahead in commits)... The event listener takes a fn as second arg, so the brackets at Line 91 should be closed.. just a gentle reminder :slightly_smiling_face:

Reply:
Oh I see @rohancl checked that in the later commit, nevermind :smile: However, when I did write that code, the server did not catch that request.. I think the "data:" token in the angular version (main Bassa) needs a corresponding token, but I could not find that in jquery so I just wrote this for partial use.. Any solutions @malithsen @vivonk @agentmilindu ?